test/functional/eval/api_functions_spec.lua

```diff
@@ -1,11 +1,33 @@
 local helpers = require('test.functional.helpers')(after_each)
 local Screen = require('test.functional.ui.screen')
 local lfs = require('lfs')
+local mpack = require('mpack')
 local neq, eq, execute = helpers.neq, helpers.eq, helpers.execute
 local clear, curbufmeths = helpers.clear, helpers.curbufmeths
 local exc_exec, expect, eval = helpers.exc_exec, helpers.expect, helpers.eval
 local insert = helpers.insert
 
+local read_mpack_file = function(fname)
+  local fd = io.open(fname, 'rb')
+  local data = fd:read('*a')
+  fd:close()
+  local unpack = mpack.Unpacker()
+  return unpack(data)
+end
+
+-- ignore metadata in API function spec
+local remove_function_metadata = function(fspec)
+  fspec['can_fail'] = nil
```

Comment:
Is it really worth keeping 0.mpack then if we just ignore the differences to 1.mpack?  Seems better then to just test at release of level 1 that no changes are done to the level-0 methods, in they keys we decided to keep. The fact that these are present in 1.mpack should be enough to fail the test if the methods are later changed in level-2 (or removed without bumping `compatibility`).
We are considering it an allowed breaking change if some client relied on the now removed keys from the 0.1.5 metadata. So level 0 does not really mean "the api/metadata as released at 0.1.5", rather it simply means all the methods that were deprecated already at the first well-specified`*` and versioned release of the api metadata, which is level 1. 

`*` A remaining TODO is of course to add some documentation for the keys to the docs.